### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by HyakuzaO** [georgedouzas/sports-betting#58] [Q&A]
Hi guys, just a question because im noob here, can we use the config file original to get results? The script will work?

**Comment by georgedouzas** [comment]
Hi,

The provided script's results (value bets and backtesting) are included in the README file (I will try to keep it updated from now on). Therefore, you can first check the backtesting results, and if you are happy, use the value bets for each betting category (i.e., you place bets only if your bookmaker offers odds higher than the ones in the value bets table).  

Remember that the backtesting results use the maximum market odds; therefore, your total return will be lower (if your bookmakers provide lower odds).

**Comment by HyakuzaO** [comment]
![imagem](https://github.com/georgedouzas/sports-betting/assets/160853868/aa1bda41-1382-44ea-ae14-668f57252056)
im trying to do myself, but, in this part what is suposed to do? im sorry for rookie level :( but im trying running this by myself and my level of python scripts etc is lower for now :( , ty if anyone help me

**Reply by georgedouzas** [reply]
To use `sports-betting`, you must be familiar with Python and machine learning. I plan to develop a GUI with plug-play models.
